sklearn/utils/extmath.py

```diff
@@ -833,3 +833,13 @@ def _deterministic_vector_sign_flip(u):
     signs = np.sign(u[range(u.shape[0]), max_abs_rows])
     u *= signs[:, np.newaxis]
     return u
+
+
+def weighted_median(array, sample_weight):
+    sorted_idx = np.argsort(array)
```

Comment:
Since it is public API, should I change it behavior to accept ``None`` for  ``sample_weight`` too. Also should I link it from the [classes.rst](https://github.com/scikit-learn/scikit-learn/blob/master/doc/modules/classes.rst) and [utilities.rst](https://github.com/scikit-learn/scikit-learn/blob/master/doc/developers/utilities.rst) ?